bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Voice.java

```diff
@@ -180,7 +181,7 @@ public static String interpret(@ParamDoc(name = "text") Object text,
      * If sink parameter is null, the error message is simply not played.
      *
      * @param text The text to interpret
-     * @param interpreter The Human Language Interpreter to be used
+     * @param interpreter Comma separated list of human language text interpreters to use or null to use the default services
```

Comment:
Please rename the parameter into `interpreters`
This parameter should remain non null. So please remove the last part of the description.